OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -287,14 +303,24 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		abstract class MovePart : Activity
+		public abstract class MovePart : Activity
 		{
-			protected readonly Move Move;
+			public readonly Move Move;
```

Comment:
`DrawLineToTarget` doesn't want or need to care about this.  Just change `GetTargets` here to `yield break`.